examples/muranoauth.yaml

```diff
@@ -0,0 +1,78 @@
+swagger: "2.0"
+
+################################################################################
+#                           Service Metadata                                   #
+################################################################################
+info:
+  version: "1.0"
+  title: Signed Request Example
+  description: This example showcase Murano authenticated requests to https://requestbin.com/r/en0xqdd251y7a .
+  contact:
+    name: maintainer name
+    email: maintainer@email.com
+
+################################################################################
+#                            API Information                                   #
+################################################################################
+
+# A hostname with a valid, signed, and active TSL certificate is required.
+host: en0xqdd251y7a.x.pipedream.net
+basePath: /
+
+x-exosite-init: sign
+x-exosite-update: signBody
+x-exosite-gc: clientCA
+
+################################################################################
+#                                  Security                                    #
+################################################################################
+securityDefinitions:
+  signature:
```

Comment:
maybe also include `basic`, `bearer` and `apiKey` here since we've defined them too above?

Reply:
```suggestion
  basic
    type: basic
    x-exosite-user-field: user
    x-exosite-secret-field: secret
  apiKey
    type: apiKey
    name: secretHeader
    in: header
    x-exosite-prefix: Token
    x-exosite-from: secret
  bearer:
    type: bearer
    name: secret
  signature:
```